cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/LoadingLimitsMapping.java

```diff
@@ -34,17 +32,12 @@ public class LoadingLimitsMapping {
     void addAll() {
         for (Map.Entry<String, LoadingLimitsAdder<?, ?>> entry : adders.entrySet()) {
             if (!Double.isNaN(entry.getValue().getPermanentLimit()) || entry.getValue().hasTemporaryLimits()) {
-                LoadingLimits limits = entry.getValue().add();
-                if (Double.isNaN(limits.getPermanentLimit())) {
-                    double fixedPermanentLimit = Iterables.get(limits.getTemporaryLimits(), 0).getValue();
-                    context.fixed("Operational Limit Set of " + entry.getKey(),
-                            "An operational limit set without permanent limit is considered with permanent limit" +
-                                    "equal to lowest TATL value",
-                            Double.NaN, fixedPermanentLimit);
-                    limits.setPermanentLimit(fixedPermanentLimit);
-                }
+                entry.getValue()
+                        .fixLimits(context.config().getMissingPermanentLimitPercentage(), context::fixed)
+                        .add();
             }
         }
         adders.clear();
     }
+
```

Comment:
Just for me: what is the rule? A space or no space at the end?

Reply:
I don't think there is a convention about this.
This empty line was added because a new method was created then removed in the successive commits of the PR.